projects/plugins/jetpack/extensions/blocks/business-hours/block.json

```diff
@@ -1,10 +1,10 @@
 {
 	"$schema": "https://schemas.wp.org/trunk/block.json",
-	"apiVersion": 3,
+	"apiVersion": 1,
```

Comment:
I believe the implementation of the `edit` function required the first version of the API.